pkg/reconciler/v1alpha1/revision/resources/deploy.go

```diff
@@ -109,6 +98,19 @@ func rewriteUserProbe(p *corev1.Probe) {
 	}
 }
 
+func getFluentdConfigMapVolume(name string) *corev1.Volume {
```

Comment:
Can we move this to `fluentd.go` and name it `makeFluentdConfigMapVolume` so it's very similar to `makeFluentdContainer`.

For the same reason, what do you think about making this function take a Revision and pull the `names.FluentdConfigMap(rev)` call inside the function? IMO that makes the signature more readable and documenting itself.

Reply:
Yeah that makes a lot of sense to me. Thanks for the suggestion.